ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -968,6 +1009,9 @@ export default class SecurityTab extends PureComponent {
         <div className="settings-page__content-padded">
           {this.renderCurrencyRateCheckToggle()}
           {this.renderIncomingTransactionsOptIn()}
+          {/* ///: BEGIN:ONLY_INCLUDE_IF(transaction-simulation) */}
+          {this.renderSimulationsToggle()}
```

Comment:
In addition to the privacy settings, I think Barbara requested we add this to the onboarding settings also.

Which could be in: `ui/pages/onboarding-flow/privacy-settings/privacy-settings.js`?

Reply:
It totally slipped my mind after implementing onboarding pages, added that in https://github.com/MetaMask/metamask-extension/pull/23604/commits/c775aae0988b548cfd88d771e552801270597613